draft-scenarios/metadata.tex

```diff
@@ -472,6 +472,8 @@
 
 \include{campaigns/main.tex}
 
+\include{coop/main.tex}
```

Comment:
Please maintain the ordering of the original. Coops  go before clash. Also, name the dir `coops`, like in the main book.